### DISCUSSION TITLE
Problem when using world vocoder for mgc extraction

### DISCUSSION THREAD

**Discussion by beqabeqa473** [RHVoice/RHVoice#471] [Q&A]
Hello.

I've encountered a problem when using World vocoder for mgc extraction.

While it really improves voice quality and more natural low frequencies are added to resulting voice, for some reason dc offset appears from somewhere and in resulting files generated with synth command very unpleasant clipping can be heared at the end of some phrases.

Could you please give an advice from could be done to eliminate this kind of issue with using new extraction method?

Thanks.

**Comment by Olga-Yakovleva** [comment]
Have your recordings been post-processed in anyway? I saw this after 
some heavy processing. I'm not an audio expert to say which specific 
kind of filtering can be a problem and why. But try on the raw 
unprocessed recordings and see if the problem remains.

**Comment by beqabeqa473** [comment]
Yes, our sound engineer said there was compressor used.

I will try to retrain using raw recorded sound files.

Thanks.

On 2/7/22, Olga Yakovleva ***@***.***> wrote:
> Have your recordings been post-processed in anyway? I saw this after
> some heavy processing. I'm not an audio expert to say which specific
> kind of filtering can be a problem and why. But try on the raw
> unprocessed recordings and see if the problem remains.
>
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/RHVoice/RHVoice/discussions/471#discussioncomment-2127207
> You are receiving this because you authored the thread.
>
> Message ID:
> ***@***.***>


-- 
with best regards Beqa Gozalishvili
Tell: +995593454005
Email: ***@***.***
Web: https://gozaltech.org
Skype: beqabeqa473
Telegram: https://t.me/gozaltech
facebook: https://facebook.com/gozaltech
twitter: https://twitter.com/beqabeqa473
Instagram: https://instagram.com/beqa.gozalishvili
